content/lessons/chapter-3/coop-2.tsx

```diff
@@ -158,108 +158,110 @@ export default function Coop2({ lang }) {
   }, [])
 
   return (
-    <div className="fade-in flex flex-col items-center  gap-5 py-8 pl-[15px] md:pt-10 2xl:gap-8 2xl:px-[75px]">
-      <HashrateChallenge
-        totalBlocks={TOTAL_BLOCKS}
-        blockRatio={BLOCK_RATIO}
-        step={step}
-        onStepUpdate={handleStepUpdate}
-        onProtagonistUpdate={handleProtagonsitBlockUpdate}
-        onAntagonistUpdate={handleAntagonsitBlockUpdate}
-        protagonists={PROTAGONISTS}
-        antagonists={ANTAGONISTS}
-        profiles={PROFILES.map((profile, i) => (
-          <Profile
-            key={i}
-            username={profile.username}
-            avatar={profile.avatar}
-            description={profile.description}
-            className={'md:!max-w-[220px]'}
+    <div className="px-[20px] py-[30px] md:py-[75px]">
```

Comment:
With the addition of the right-padding here the overflow cutoff has returned for the profiles instead of at the edge of the screen they are cutof at the padding edge.

Reply:
Yes, I added the back in so the scrollbar doesn't touch the edges and look weird (you can't tell if it's cut off or not). Ideally, we could control the position of the scrollbar itself. But since that's not possible, I think this is a good compromise.